textalyze.js

```diff
@@ -57,19 +69,16 @@ if (require.main === module) {
   // const printCounts = counts => console.table(counts);
   // const program = compose(printCounts, itemCounts, stringToCharacters, sanitize);
 
-  const program = x => console.table(itemCounts(stringToCharacters(sanitize(x))));
+  const program = x => printHistogram(itemFrequencyCounter(stringToCharacters(sanitize(x))));
```

Comment:
Nice!

Reply:
I wouldn't call it `program`, even those that's what I did in the example I gave.  I just wanted to show that this "program" is really a composition of several simple functions.  Give it a name like `printFileHistogram` and call `x` something more descriptive like `file` or `path` or `fileName`.